openassessment/xblock/grade_mixin.py

```diff
@@ -130,6 +130,49 @@ def render_grade_complete(self, workflow):
         # when all the criteria in the rubric are feedback-only (no options).
         score = workflow['score']
 
+        first_sentence = "You have successfully completed this problem and received a {}/{}.".format(
+            score["points_earned"], score["points_possible"]
+        )
+        second_sentence = ""
```

Comment:
Can you move this block of code to a separate function and pass `assessment_steps` as an argument?
Something like `_get_score_explanation()`.